tests/astrometry_net_data/photocal/andConfig.py

```diff
@@ -0,0 +1,7 @@
+root.defaultMagColumn = 'r'
+root.starGalaxyColumn = 'star'
+#root.variableColumn = 'variable'
```

Comment:
Never mind. You were right about the andconfig files. Only one of them (the one with no digit in the name) was wanted. I'll remove the others and remove the one commented-out line from the remaining file.